app/controllers/issues_controller.rb

```diff
@@ -34,6 +34,11 @@ def popup
                  img: @issue.first_attached_image)
   end
 
+  def closed
```

Comment:
This action duplicate #index action except scope on Issue, so why do not you pass it as parameter and use single action for both cases, when you need approved or closed issues?